profile/README.md

```diff
@@ -0,0 +1,38 @@
+# Welcome to Kwil!
+
+![Masthead](../images/welcome-kwil.gif)
+
+----
+<p align="center">
+<i>Welcome to the Kwil Github Organization!</i>
+</p>
+
+-----
+
+
+Kwil is a framework for leveraging relational databases as standalone, byzantine fault tolerant networks. Kwil extends the functionality of traditional relational databases by providing a practical byanztine fault tolerant (pBFT) layer on top of existing engines, enforcing rules necessary for building secure, decentralized data stores.
+
+## Build
+
+Below are popular repositories for building with Kwil:
+
+| Name | Description | Link |
+|----------|----------|----------|
+| binary-releases | Release repository for latest Kwil Builds & CLIs | [Here](https://github.com/kwilteam/binary-releases) |
+| kwil-db | Golang implementation of Kwil Daemon | Open Source in Q4 2023 |
+| kwil-js | JavaScript/TypeScript SDK for Kwil Applications | [Here](https://github.com/kwilteam/kwil-js) |
+| kwil.py | Python SDK for Kwil Applications | [Here](https://github.com/kwilteam/kwil.py) |
+| kuneiform |  Language for defining databases on Kwil | [Here](https://github.com/kwilteam/kuneiform) |
+
+## Contributing
+
+Kwil welcomes pull requests on our public repositories. Please follow each repository's contributing guidelines.
+
+Open full time roles at Kwil are posted on our [careers page](https://www.kwil.com/careers).
+
+## Resources
+
+- 💻 [Website](https://www.kwil.com)
```

Comment:
Yes or no on emojis?

Reply:
I would move away from emojis - Vanesa made a good point that they don't add too much in the form of context with the downside of looking less refined

Reply:
I'd vote without.  Everything else looks goos